shared/templates/template_OVAL_shell_lineinfile

```diff
@@ -0,0 +1,10 @@
+{{{
+oval_check_shell_file(
+	path=PATH,
+	parameter=PARAMETER,
+	value=VALUE,
+	no_quotes=NO_QUOTES,
+	missing_parameter_pass=MISSING_PARAMETER_PASS
```

Comment:
I think `application` should be set otherwise it generates incomplete comments in OVAL, eg. `<criteria comment=" is configured correctly" operator="OR">`. I don't know what value set it to, though.